app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -641,6 +641,17 @@
     <div class="field">{{i18n "user.invited.days_visited"}}</div>
     <div class="value">{{html-safe model.days_visited}}</div>
   </div>
+  <div class="display-row post-edits-count">
+    <div class="field">{{i18n "user.invited.post_edits_count" }}</div>
+    <div class="value">
+      {{html-safe model.post_edits_count}}
+      {{d-button class="widget-button btn-flat heatmap-high btn-icon"
+        action=(action "showPostEditsModal")
+        icon="far-eye"
```

Comment:
My preference would be for this to use a gray button that is over to the right like the other fields on the admin page like "Show Flags Received". I know this doesn't match the spec we sent over but seeing it in the screenshots I would prefer it to be consistent with the other admin controls.